oms/src/main/java/com/gfg/oms/TestApp.java

```diff
@@ -0,0 +1,40 @@
+package com.gfg.oms;
+
+import com.gfg.oms.client.Client;
+import com.gfg.oms.consignment.request.Attribute;
+import com.gfg.oms.consignment.request.CreateCommand;
+import com.gfg.oms.consignment.request.Item;
+import com.gfg.oms.consignment.response.CreateConsignmentResponse;
+
+import java.io.IOException;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class TestApp {
+    public static void main(String[] args) throws IOException {
+        Client client = new Client(new Config("http://oms-mock.devbox.local/oms/api","blabla"));
```

Comment:
Is this requires real connection to oms-mock? Do you think this is needed?

Reply:
No, this is throw-away example app to demonstrate how this library will be used. Of course, this class will be removed after review.